project/Dependencies.scala

```diff
@@ -33,7 +31,11 @@ object Dependencies {
   val akkaTestKit: ModuleID =           "com.typesafe.akka" %% "akka-testkit"             % akkaV     % "test"
   val akkaHttpTestKit: ModuleID =       "com.typesafe.akka" %% "akka-http-testkit"        % akkaHttpV % "test"
 
-  val cromwellClient: ModuleID =    "org.broadinstitute.cromwell" %% "cromwell-client" % "0.1-8b413b45f-SNAP"
+  // This version of `cromwell-client` was packaged by `sbt` running on Scala 2.12 but actually contains only Java
+  // classes (generated from OpenAPI YAML) that were not compiled against any version of the Scala library.
+  // `cromwell-client` is therefore referenced here as a Java artifact with "_2.12" incorporated into its name,
+  // allowing for Rawls to upgrade its Scala version without requiring any changes to this artifact.
```

Comment:
This comment is confusing to me. Why does labeling the artifact with `_2.12` allow us to upgrade Rawls? Does this just mean that we can upgrade Rawls despite the 2.12 reference here? 

IIRC we didn't change this to 2.13 because it caused some tests to break, that might be worth noting here if we're planning to leave this at 2.12 after the upgrade to 2.13.

Reply:
I think cromwell client really shouldn't be labeled either 2.12 or 2.13...it's just a java lib, hence it doesn't matter what version Rawls is on, it can use it just like using any other java lib